cmd/policy-assistant/pkg/kube/ikubernetes.go

```diff
@@ -28,6 +30,16 @@ type IKubernetes interface {
 	DeleteService(namespace string, name string) error
 	GetServicesInNamespace(namespace string) ([]v1.Service, error)
 
+	GetAdminNetworkPolicies(ctx context.Context) ([]v1alpha1.AdminNetworkPolicy, error)
+	CreateAdminNetworkPolicy(ctx context.Context, policy *v1alpha1.AdminNetworkPolicy) (*v1alpha1.AdminNetworkPolicy, error)
+	UpdateAdminNetworkPolicy(ctx context.Context, policy *v1alpha1.AdminNetworkPolicy) (*v1alpha1.AdminNetworkPolicy, error)
+	DeleteAdminNetworkPolicy(ctx context.Context, name string) error
+
+	GetBaselineAdminNetworkPolicies(ctx context.Context) ([]v1alpha1.BaselineAdminNetworkPolicy, error)
```

Comment:
nit: similar to https://github.com/kubernetes-sigs/network-policy-api/pull/239#discussion_r1717400729, I would make this and all other BANP functions singular:
```suggestion
	GetBaselineAdminNetworkPolicy(ctx context.Context) (v1alpha1.BaselineAdminNetworkPolicy, error)
```